docs/G-Codes.md

```diff
@@ -1006,6 +1006,13 @@ appropriate DRIVE_CURRENT for the sensor. After running this command
 use the SAVE_CONFIG command to store that new setting in the
 printer.cfg config file.
 
+#### SET_PROBE_DRIFT_ADJ_FACTOR
+`SET_PROBE_DRIFT_ADJ_FACTOR PROBE=<config_name> FACTOR=<new_factor>`:
+When thermal drift compensation is calibrated this command may be
+used to set the `drift_adj_factor` during runtime.  This value
+is a multiplier applied to the calculated drift compensation and may be
+used to fine tune the result.
```

Comment:
For what it is worth, this command seems very subtle to me.  Might be better to hold off on it until we know it is needed.  Similar with the config option.

Or another alternative might be to have something like a `PROBE_TEMPERATURE_ENABLE` command to let users manually enable/disable temperature_probe without the subtle fine tuning.

Reply:
Sounds good.